messaging/src/main/java/org/axonframework/queryhandling/QueryUpdateEmitter.java

```diff
@@ -162,10 +162,24 @@
      * @param <U>              the incremental response types of the query
      * @return the object which contains updates and a registration which can be used to cancel them
      */
+    @Deprecated
     <U> UpdateHandlerRegistration<U> registerUpdateHandler(SubscriptionQueryMessage<?, ?, ?> query,
                                                            SubscriptionQueryBackpressure backpressure,
                                                            int updateBufferSize);
 
+    /**
+     * Registers an Update Handler for given {@code query} with given {@code backpressure} and {@code
```

Comment:
`backpressure` is no longer present in this method, so should be removed from the Javadoc too.